src/core/web_api/p2_api.js

```diff
@@ -58,14 +58,29 @@ class Auth0APIClient {
   }
 
   logIn(options, authParams, cb) {
+    debugger;
     // TODO: for passwordless only, try to clean in auth0.js
     // client._shouldRedirect = redirect || responseType === "code" || !!redirectUrl;
     const f = loginCallback(false, this.domain, cb);
-    const loginOptions = normalizeAuthParams({ ...options, ...this.authOpt, ...authParams });
+    const loginOptions = normalizeAuthParams({
+      ...options,
+      ...this.authOpt,
+      ...authParams
+    });
+
+    if (options.login_hint) {
+      loginOptions.login_hint = options.login_hint;
```

Comment:
please add a test for this. file: https://github.com/auth0/lock/blob/master/src/__tests__/core/web_api/p2_api.test.js#L105